docs/cli-reference/start.md

```diff
@@ -68,7 +68,7 @@ ollama_library_base_url: https://registry.mycompany.com
 # Worker Options
 server_url: http://myserver
 worker_ip: 192.168.1.101
-enable_metrics: true
```

Comment:
--enable-metrics in this file also need to update, line 40

Reply:
Updated